sklearn/utils/testing.py

```diff
@@ -615,10 +609,10 @@ def clean_warning_registry():
     https://bugs.python.org/issue21724 for more details.
```

Comment:
You could update the docstring about (the Python 2 part)

Reply:
Thanks. Fixed.